src/modal/sign-txn/PeraWalletSignTxnModal.tsx

```diff
@@ -0,0 +1,40 @@
+import "../_pera-wallet-modal.scss";
+import "./_pera-wallet-sign-txn-modal.scss";
+
+import PeraConnectIcon from "../../asset/icon/PeraConnect.svg";
+import CloseIcon from "../../asset/icon/Close.svg";
+
+import React from "react";
+
+interface PeraWalletSignTxnModalProps {
+  onClose: VoidFunction;
+}
+
+function PeraWalletSignTxnModal({onClose}: PeraWalletSignTxnModalProps) {
+  return (
+    <div className={"pera-wallet-modal pera-wallet-sign-txn-modal"}>
+      <div className={"pera-wallet-modal__body"}>
+        <div className={"pera-wallet-modal__body__header"}>
```

Comment:
Maybe we can make the Header a separate component it's also used in `PeraWalletConnectModal`